docs/index.md

```diff
@@ -49,7 +49,7 @@ We think {abbr}`MyST (Markedly Structured Text)` is really cool, some of the fea
 :::{seealso}
 :class: dropdown
 
-# Coming from Jupyter Book or Sphinx?
+# Coming from Jupyter Book V1 or Sphinx?
```

Comment:
I think this could be expanded (in this PR on in another when we are in beta perhaps?). We should link to the blog article and say that jupyterbook v2 will be a distribution of mystmd.